README.rst

```diff
@@ -273,6 +273,8 @@ on the first line and urls or paths that are on their own line::
       "https://some-super-long-domain-name.com/with/some/very/long/paths"
   )
 
+**B910**: Use Counter() instead of defaultdict(int) to avoid memory leaks. Using the latter can result in using as much as ~1000 MB of memory even for small and simple dicts. Refactoring the code to use the former can reduce memory usage from mentioned ~1000 MB to flat ~20 MB.
```

Comment:
Where does this number come from? I would expect it depends on the size of the data; i.e., you can construct a Counter that is a 1000 MB and also a defaultdict that is 20 MB.

It appears the difference is that when you access a key that does not exist, Counter simply returns 0, but defaultdict sets the value to 0 in the dict as well. How much that affects your memory usage obviously depends on how often you access keys that don't exist. We should mention the pattern that leads to the difference, not the exact number, because the exact number is going to depend on your use case.

Reply:
I think the numbers were based on the test case I provided in #323, and yes they will depend on the number of missing keys accessed.

How about: "Use Counter() instead of defaultdict(int) to avoid excessive memory use as the default dict will record missing keys with the default value when accessed."

Reply:
Yes, that sounds good.

Reply:
+1 - Thanks all for discussing!

Reply:
Thanks, I agree that it wasn't the best decision to mention any numbers there. I changed that.